packages/core/strapi/lib/core/domain/content-type/validator.js

```diff
@@ -17,15 +17,23 @@ const contentTypeSchemaValidator = yup.object().shape({
     info: yup
       .object()
       .shape({
+        displayName: yup.string().required(),
         singularName: yup
           .string()
           .isKebabCase()
           .required(),
         pluralName: yup
           .string()
           .isKebabCase()
-          .required(),
-        displayName: yup.string().required(),
+          .test(
```

Comment:
We need the plural for a single type too (database table name must use the plural) :) and since we can switch from one kind ot the other that will be simpler to handle too

Reply:
oki :)